src/features/defi/providers/idle/components/IdleManager/Overview/IdleEmpty.tsx

```diff
@@ -0,0 +1,218 @@
+import {
+  Alert,
+  Button,
+  Flex,
+  Image,
+  List,
+  ListIcon,
+  ListItem,
+  Stack,
+  Tag,
+  useColorModeValue,
+} from '@chakra-ui/react'
+import type { AssetId } from '@shapeshiftoss/caip'
+import { DefiModalContent } from 'features/defi/components/DefiModal/DefiModalContent'
+import { EmptyOverview } from 'features/defi/components/EmptyOverview/EmptyOverview'
+import { useCallback, useMemo } from 'react'
+import { MdCheckCircle } from 'react-icons/md'
+import { useTranslate } from 'react-polyglot'
+import BestYieldIcon from 'assets/best-yield.svg'
+import IdleBg from 'assets/idle-bg.png'
+import JuniorTrancheIcon from 'assets/junior-tranche.svg'
+import SeniorTranceIcon from 'assets/senior-tranche.svg'
+import { AssetIcon } from 'components/AssetIcon'
+import { Carousel } from 'components/Carousel/Carousel'
+import { FiatRampAction } from 'components/Modals/FiatRamps/FiatRampsCommon'
+import { SlideTransition } from 'components/SlideTransition'
+import { Text } from 'components/Text'
+import { useModal } from 'hooks/useModal/useModal'
+import { bnOrZero } from 'lib/bignumber/bignumber'
+import { selectSupportsFiatRampByAssetId } from 'state/apis/fiatRamps/selectors'
+import { IdleTag } from 'state/slices/opportunitiesSlice/resolvers/idle/constants'
+import type { TagDescription } from 'state/slices/opportunitiesSlice/types'
+import { selectAssetById, selectPortfolioCryptoHumanBalanceByFilter } from 'state/slices/selectors'
+import { useAppSelector } from 'state/store'
+
+const idleTagDescriptions: Record<IdleTag, TagDescription[]> = {
+  [IdleTag.BestYield]: [
+    {
+      title: 'idle.emptyBody.bestYield.page-1.title',
+      description: 'idle.emptyBody.bestYield.page-1.body',
+      icon: <Image src={BestYieldIcon} />,
+    },
+    {
+      title: 'idle.emptyBody.bestYield.page-2.title',
+      bullets: ['idle.emptyBody.monitorSystem', 'idle.emptyBody.offerHighReturn'],
+    },
+  ],
+  [IdleTag.JuniorTranche]: [
+    {
+      title: 'idle.emptyBody.juniorTranche.page-1.title',
+      description: 'idle.emptyBody.juniorTranche.page-1.body',
+      icon: <Image src={JuniorTrancheIcon} />,
+    },
+    {
+      title: 'idle.emptyBody.juniorTranche.page-2.title',
+      bullets: [
+        'idle.emptyBody.leverageYield',
+        'idle.emptyBody.higherRisk',
+        'idle.emptyBody.noLockingPeriod',
+        'idle.emptyBody.autoReinvest',
+      ],
+    },
+  ],
+  [IdleTag.SeniorTranche]: [
+    {
+      title: 'idle.emptyBody.seniorTranche.page-1.title',
+      description: 'idle.emptyBody.seniorTranche.page-1.body',
+      icon: <Image src={SeniorTranceIcon} />,
+    },
+    {
+      title: 'idle.emptyBody.seniorTranche.page-2.title',
+      bullets: [
+        'idle.emptyBody.minRisk',
+        'idle.emptyBody.builtInCoverage',
+        'idle.emptyBody.noLockingPeriod',
```

Comment:
I reckon that's the intended terminology?
```suggestion
        'idle.emptyBody.noLockinPeriod',
```

Reply:
The translation is `No locking period or epochs` so just trimmed it down to `noLockingPeriod`